.github/scripts/trymerge.py

```diff
@@ -1397,10 +1409,34 @@ def validate_land_time_checks(org: str, project: str, commit: str) -> None:
 def has_label(labels: List[str], pattern: Pattern[str] = CIFLOW_LABEL) -> bool:
     return len(list(filter(pattern.match, labels))) > 0
 
-def categorize_checks(
-    check_runs: JobNameToStateDict,
-    required_checks: List[str],
-) -> Tuple[List[Tuple[str, Optional[str]]], List[Tuple[str, Optional[str]]]]:
+def get_release_notes_labels() -> List[str]:
+    return [label for label in get_pytorch_labels() if label.lstrip().startswith("release notes:")]
+
+def has_required_labels(pr: GitHubPR) -> bool:
+    pr_labels = pr.get_labels()
+    # Check if PR is not user facing
+    is_not_user_facing_pr = any(label.strip() == "topic: not user facing" for label in pr_labels)
+    return is_not_user_facing_pr or any(label.strip() in get_release_notes_labels() for label in pr_labels)
+
+def delete_comment(comment_id: int) -> None:
+    url = f"https://api.github.com/repos/pytorch/pytorch/issues/comments/{comment_id}"
+    _fetch_url(url, method="DELETE")
```

Comment:
As commented on another function, do not make `pytorch` specific assumptions, pass org and project options
```suggestion
def gh_post_delete_comment(org: str, project: str, comment_id: int) -> None:
    url = f"https://api.github.com/repos/{org}/{project}/issues/comments/{comment_id}"
    _fetch_url(url, method="DELETE")
```